### DISCUSSION TITLE
Using --data with a multiselect based choice

### DISCUSSION THREAD

**Discussion by chrisdaish** [copier-org/copier#1473] [Q&A] [answered]
Is it possible to use --data to pass in a value to a multiselect (choices) based question?

I would have though you just passed it in like any other key=value, however the value is reported as invalid.

**Comment by sisp** [comment, marked as answer]
I'm afraid this isn't possible at the moment, but it should be. Would you mind creating an issue, @chrisdaish?

/cc @noirbizarre

**Reply by chrisdaish** [reply]
Issue created: https://github.com/copier-org/copier/issues/1474

**Reply by noirbizarre** [reply]
I am using the python version and it's just a list of the values.
It is possible that passing them through `--data` might need an extra processing to properly typecast it.
Here's the original PR if you want to dig into: https://github.com/copier-org/copier/pull/1386
